REQUIRE

```diff
@@ -1,10 +1,10 @@
-julia 0.5
+julia 0.6
```

Comment:
should stop testing on 0.5 if Gadfly is dropping support for it, particularly since METADATA no longer supports it.